libs/community/langchain_community/vectorstores/sqlserver.py

```diff
@@ -66,6 +106,11 @@ class DistanceStrategy(str, Enum):
 EXTRA_PARAMS = ";Trusted_Connection=Yes"
 INVALID_IDS_ERROR_MESSAGE = "Invalid list of ids provided"
 INVALID_INPUT_ERROR_MESSAGE = "Input is not valid."
+INVALID_FILTER_INPUT_EXPECTED_DICT = "Invalid filter condition. Expected a dictionary "
+"but got an empty dictionary"
```

Comment:
I don't think this is seen as a part of `INVALID_FILTER_INPUT_EXPECTED_DICT` const definition? You can create a multiline string for it:
"""Invalid filter condition. Expected a dictionary
but got an empty dictionary"""

Reply:
done